server/lib/src/constants/schema.rs

```diff
@@ -211,6 +211,40 @@ pub const JSON_SCHEMA_ATTR_NAME_HISTORY: &str = r#"{
     }
 }"#;
 
+pub const JSON_SCHEMA_ATTR_ECDH_KEY: &str = r#"{
+    "attrs": {
+      "class": [
+        "object",
+        "system",
+        "attributetype"
+      ],
+      "description": [
+        "Account verification public and optionally private key "
```

Comment:
Should probably just comment that it's an EC key because it could be used for other things.

Reply:
Also given it's an ECKey, it's not limited to ECDH, so name this ECKEY not ECDH.